vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -126,7 +127,8 @@ std::vector<facebook::react::NativeModuleDescription> GetModules(
   std::shared_ptr<DeviceInfo> deviceInfo,
   std::shared_ptr<facebook::react::DevSettings> devSettings,
   const I18nModule::I18nInfo&& i18nInfo,
-  std::shared_ptr<facebook::react::AppState> appstate)
+  std::shared_ptr<facebook::react::AppState> appstate,
+  std::shared_ptr<IReactInstance> nativeAnimated)
```

Comment:
>nativeAnimated [](start = 34, length = 14)

variable name of nativeAnimated seems odd, should be something like uwpInstance or spInstance